### DISCUSSION TITLE
DOSXYZnrc

### DISCUSSION THREAD

**Discussion by mona399** [nrc-cnrc/EGSnrc#1184] [General]
I have some questions:
the first question is if I need to define water phantom 30cm*30cm*30cm and determine material similar below code by  DOSXYZnrc   GUI how can I determine PEGS4?
# Phantom description in cm

# Water phantom (up to depth 10 cm)
Region 1: 
   Material: Water
   Z-boundary: 0.0 to 10.0 cm

# Top polyester layer (125 µm)
Region 2:
   Material: Polyester
   Z-boundary: 10.0 cm to 10.0125 cm

# Active layer (28 µm)
Region 3:
   Material: Active Layer
   Z-boundary: 10.0125 cm to 10.0153 cm

# Bottom polyester layer (125 µm)
Region 4:
   Material: Polyester
   Z-boundary: 10.0153 cm to 10.0278 cm

# Water phantom (after the film)
Region 5:
   Material: Water
   Z-boundary: 10.0278 cm to the end
the second question is When selecting the voxel to know the dose values ​​on the z-axis and then viewing the results using the Victoria program and then importing dose profiles to CSV, what are the most appropriate dose values ​​to take at a specific depth, The dose values ​​are in the CSV file or the egslst file. For more clarification, see the images below
![image](https://github.com/user-attachments/assets/343207bd-4923-4669-8569-e6f44e8db15e)
![image](https://github.com/user-attachments/assets/59b9399c-7b9f-4131-a2f6-afe9ecac8614)
![image](https://github.com/user-attachments/assets/38c57763-ccf4-4a40-b261-39e6e17d4de7)

can  anyone help me, please

**Comment by rtownson** [comment]
I'm not really sure what you're asking, can you clarify? If you're asking about how to define a material, there are some defaults in the 700icru.pegs4dat and 521icru.pegs4dat material pegs4 files. To create your own material, you can use egs_gui, or you can follow https://github.com/nrc-cnrc/EGSnrc/wiki/Generating-density-effect-correction-files-with-ESTAR and then define all of your materials using density correction files ("pegsless"). 

For viewing dose in VICTORIA and getting profiles from it, you don't need to use `dosxyz dose outputs`. Instead, take the 3ddose file that DOSXYZnrc produces, and load that in VICTORIA. Then it can extract profiles and depth dose curves from the 3ddose data. This would be equivalent to what you're already doing using `dosxyz dose outputs`. 

In terms of "what are the most appropriate dose values ​​to take at a specific depth", are you asking how to interpret the table of dose values? It's just giving the list of dose in the voxels at the positions specified by the x,y,z boundaries, and the errors on each dose.

**Comment by mona399** [comment]
Thank you for your reply. To clarify the first question, I want to build a
phantom of water and put a material similar to the properties of
radiochromic film at a depth of 10 cm. As for the second question, I want
to find the dose value at a depth of 10 cm to use it when calculating the
Output factor. Can you explain to me the steps that must be followed after
running in DOSXZYnrc?

**Reply by rtownson** [reply]
To get the dose at 10cm on the central axis, import the 3ddose file into VICTORIA. Turn on the dose profile option, and click your cursor on the appropriate location (it displays the coordinates). You can get the dose directly in VICTORIA, or you can export the profile to csv. Make sure that you understand if there is one central voxel, or if you need to average the two voxels on either side of the central axis.

**Comment by mona399** [comment]
Yes I know these steps but my question is in the egslst file there is
DOSXZYnrc output dose as shown in the image above my question is where do I
use these values? Note that when using Victoria and using the 3ddose file
the dose values ​​are very similar to those in the egslst file but the
difference is that in Victoria the values ​​start from -14.5 while in the
egslst file they start from -15 note that my minimum boundary in
voxel=-15cm

**Reply by rtownson** [reply]
I imagine that VICTORIA is reporting the center of the voxel as the position.

**Reply by mona399** [reply]
yes.in VICTORIA give value in bins for example -14.5 cm represent the center of the first bin, which collects all value from -15cm to -14cm

**Comment by serapcatli** [comment]
Hi everyone, My name is Serap. I have a question. I created a dosxynznrc file. It compiles but it does not run. If I send you the file, can you give me information about why it is not working? Thank you.

**Reply by rtownson** [reply]
Please create a new discussion for your question, and attach the .egsinp file after renaming it to .txt (because .egsinp files are not supported).

Also include as much information as possible, like the errors that you see in the output. Screenshots help, etc.

**Comment by serapcatli** [comment]
I can't send it here because it has an egsinp extension. I can send it by e-mail if you want?

**Comment by kouadriaek** [comment]
hi
how to do parallel job ex field 10x10 cm 1nd 20x20 cm in same time

**Reply by rtownson** [reply]
Please create a new discussion for your question. Include as much information as possible, like what you've tried so far and the errors that you see in the output. Screenshots help, etc.
